packages/select/Select.tsx

```diff
@@ -0,0 +1,116 @@
+import { h, Component } from 'skatejs';
+import { GenericEvents, prop } from '../_helpers';
+import Option from './Option';
+import { bind } from 'decko';
+import { customElement, shadyCssStyles } from '../_helpers/decorators';
+import { SelectOverlay } from './components/Overlay';
+import styles from './Select.scss';
+import { SelectButton } from './components/Button';
+import { SelectCard } from './components/Card';
+
+export type SelectProps = Props & EventHandlers;
+
+export type Props = {
+  placeholder?: string,
+  value?: any,
+  open?: boolean,
+};
+
+export type Events = {
+  click?: GenericEvents.ClickHandler,
+};
+export type EventHandlers = {
+  onClick?: GenericEvents.ClickHandler,
+};
+
+@shadyCssStyles()
+@customElement( 'bl-select' )
+export default class Select extends Component<SelectProps> {
+
+  @prop( { type: String } ) placeholder: string;
+  @prop( { type: Boolean, attribute: { source: true } } ) open = true;
+  @prop( { attribute: true } ) value: any;
```

Comment:
do we really need 2 way reflection to attributes? what if value is object or so?